examples/basic/variables.tf

```diff
@@ -33,3 +33,9 @@ variable "resource_tags" {
   description = "Optional list of tags to be added to created resources"
   default     = []
 }
+
+variable "enable_cors" {
```

Comment:
Do not expose this in the example. We don't want example to become flexible solutions. Leave this out of the basic example so it defaults to the default value of the module. Keep the basic example simple